### PR TITLE
Added changes to README.md related to the azure-mgmt-postgresqlflexibleservers library

### PR DESCRIPTION
**Pull request by gkasar**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by gkasar** [comment]
Please review this file @gbowerman

**Review by gbowerman — APPROVED** [review]

**Comment by gbowerman** [comment]
#sign-off

File: sdk/postgresqlflexibleservers/azure-mgmt-postgresqlflexibleservers/README.md [hunk 1]
```diff
@@ -9,14 +9,16 @@ For a more complete view of Azure libraries, see the [azure sdk python release](
 _Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
 
 ## Getting started
+This library is used to provision an Azure Database for PostgreSQL flexible server instance, multiple servers, or multiple databases on a server.
```
[the remark this answers is not in the document]
Comment (gkasar) [reply sdk/postgresqlflexibleservers/azure-mgmt-postgresqlflexibleservers/README.md:12]: Hi @msyyc 

Thank you for this! I want to know what is the expected timeline to get this PR merged/released https://github.com/Azure/azure-sdk-for-python/pull/36805/files#diff-621af6b33ab5945420d92610c5f56e68ba4c8f62991e1b4fb6d45cf9b7431b88

[the remark this answers is not in the document]
Comment (msyyc) [reply sdk/postgresqlflexibleservers/azure-mgmt-postgresqlflexibleservers/README.md:12]: We hope to release it by the end of this week.

**Comment by annatisch** [comment]
Looks like this has been addressed - I'm going to close but please let us know if I should reopen for any reason :)

**Pull request closed without merging**